lib/core/Proxy.js

```diff
@@ -64,9 +71,74 @@ class Proxy {
       headers: request.headers,
     };
     logger.info(`${HTTPLOG} Received http request ${options}`);
-    if(request.url.indexOf('/status') > -1){
-      response.writeHead(200, {'content-type': 'application/json; charset=utf-8', 'accept': 'application/json', 'WWW-Authenticate': 'Basic realm="WS Reconnect Proxy"'});
-      response.end(JSON.stringify({"status" : "Running"}));
+    if (request.url.indexOf('/status') > -1) {
+      response.writeHead(200, {
+        'content-type': 'application/json; charset=utf-8',
+        accept: 'application/json',
+        'WWW-Authenticate': 'Basic realm="WS Reconnect Proxy"',
+      });
```

Comment:
Since we are using the same object later in the code.Let's make it a constant and reuse it wherever required.